static/specification.html

```diff
@@ -514,210 +271,189 @@ <h2>
             </tr>
             <tr>
               <td>
-                <code><dfn>start</dfn></code>
+                <code><dfn data-dfn-for="Monetization" class=
+                "event">start</dfn></code>
               </td>
               <td>
-                {{ MonetizationStartEvent }}
+                {{MonetizationEvent}}
               </td>
               <td>
                 The first non-zero payment is successfully sent.
               </td>
-              <td>{{ Document / monetization }} attribute</td>
+              <td>
+                {{Navigator/monetization}} attribute
+              </td>
             </tr>
             <tr>
               <td>
-                <code><dfn>progress</dfn></code>
+                <code><dfn data-dfn-for="monetization" class=
+                "event">progress</dfn></code>
               </td>
               <td>
-                {{ MonetizationProgressEvent }}
+                {{MonetizationEvent}}
               </td>
               <td>
                 A payment is successfully made to the receiver.
               </td>
               <td>
-                {{ Document / monetization }}
+                {{Navigator/monetization}}
               </td>
             </tr>
           </table>
         </section>
-        <section data-dfn-for="MonetizationStartEvent">
-          <h2><dfn>MonetizationStartEvent</dfn> interface</h2>
+        <section data-dfn-for="MonetizationEvent">
+          <h2>
+            `MonetizationEvent` interface
+          </h2>
           <pre class="idl">
             [SecureContext, Exposed=Window]
-            interface MonetizationStartEvent : Event {
-              constructor(DOMString type, MonetizationStartEventInit eventInitDict);
-              readonly attribute DOMString paymentPointer;
+            interface MonetizationEvent : Event {
+              constructor(DOMString type, MonetizationEventInit eventInitDict);
+              readonly attribute USVString paymentPointer;
```

Comment:
if we're no longer using the payment pointer format and using URL format instead, it might be better to call this something like `receiver`

Reply:
Would this be the ILP address, or the SPSP endpoint?

Reply:
Ok, which one is it going to be? 🙏

Reply:
@sharafian isn't it still a payment pointer even if it's in the URL form?

Reply:
Related https://github.com/WICG/webmonetization/issues/201

Reply:
I think I'm going to switch this to `url`. It's also what @sidvishnoi did in his implementation.

Reply:
> if we're no longer using the payment pointer format and using URL format instead, it might be better to call this something like receiver

Reply:
Payment pointers are referenced extensively in Open API   
https://github.com/interledger/open-payments/blob/main/open-api-spec.yaml